### DISCUSSION TITLE
OnClick get the material texture X and Y

### DISCUSSION THREAD

**Discussion by fmercerat** [google/model-viewer#3827] [Ideas]
It would be nice that in the event you can get the x and the y coordinates of the 2d texture where the mouse clicked.

**Comment by elalish** [comment]
Sounds good, would you like to contribute a PR?

**Reply by fmercerat** [reply]
Im working with positionAndNormalFromPoint(). As soon as i make the script ill do a PR. 
Im thinking of getting the uv then the material from materialFromPoint() and then divide the pixels from the image with the uv factor. But i dont know which way is the faster to get the pixels of the texture image
